src/holofuel/components/HashIcon/HashIcon.js

```diff
@@ -13,7 +13,8 @@ export default function HashIcon ({
       seed={hash}
       size={size}
       className={className}
-      circle />
+      circle
+    />
```

Comment:
really don't want us to do this if we can avoid it. I think this is a standardjs version thing. Your linter might be a newer version of standard than the one in the repo

Reply:
standard 13.1.0 does the trick.